examples/next-app-router/components/article.tsx

```diff
@@ -56,16 +55,6 @@ export const Blog = ({ blog }: { blog: BlogProps }) => {
                 fieldId: 'file',
               })}
             />
-            <div className="space-y-4 md:space-y-6">
```

Comment:
why did we remove the rich text here?

Reply:
It's not working with React 19 yet, see this PR: https://github.com/contentful/rich-text/pull/756
I would bring it back once we added support there (currently blocked because of a Jest issue)